netlify/remix-compute
# chore: added vitest to the project

## Description

Adds the [vitest](https://vitest.dev/) test framweork to the project.

## Related Tickets & Documents



- Related Issue #
- Closes #8

## QA Instructions, Screenshots, Recordings

1. Pull down this branch, e.g. via the GitHub CLI, `gh co 13`
2. Run npm install
3. Run npm test
4. One test passes.

<img width="787" alt="CleanShot 2023-01-25 at 09 06 21@2x" src="https://user-images.githubusercontent.com/833231/214584577-04e1c46e-65e2-45c6-b964-da2bc3fa506e.png">

For us to review and ship your PR efficiently, please perform the following steps:

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.

**A picture of a cute animal (not mandatory, but encouraged)**

![A hamster running in a wheel then loosing control](https://media.giphy.com/media/yaUG0KDAcIcWA/giphy.gif)
